tests/api_data/endpoint_data.json

```diff
@@ -471,7 +513,13 @@
       "type_of_application": "sporting",
       "project_name": "Save the rugby pitch",
       "funding_amount_requested": "25000",
-      "location_json_blob": "Scotland"
+      "location_json_blob": {
+        "error": false,
```

Comment:
what is this error key for?

Reply:
It's what is returned from postcodes.io to let you know it's a valid postcode - means if needed in the future we can quickly filter out or identify invalid postcodes